install.sh

```diff
@@ -326,12 +254,14 @@ libsodium_install ()
 
 joinmarket_install ()
 {
-    jm_pkgs=( 'jmbase' 'jmdaemon' 'jmbitcoin' 'jmclient' )
-    for pkg in ${jm_pkgs[@]}; do
-        pip uninstall -y "${pkg/jm/joinmarket}"
-        pushd "${pkg}"
-        pip install ${develop_build:+-e} . || return 1
-        popd
+    reqs=( 'base.txt' )
+
+    if [[ -n ${with_qt} ]]; then
+        reqs+=( 'gui.txt' )
+    fi
+
+    for req in ${reqs[@]}; do
+        pip install -r "requirements/${req}" || return 1
```

Comment:
As now `-e` is in `requirements/base.txt`, always on and aren't used in `install.sh`, it should be removed (both `--develop` and `develop_build`).

Reply:
I changed my mind about this one. `--develop` should be kept for now as a parameter, just `develop_build` variable should be removed. Some people might have it in some automation scripts or whatever, instead of removing, maybe some message about `--develop` being deprecated (always on) should be printed, if it's passed.

Reply:
We decided to make regular installs use in-place editing instead of copying (which has led to issues in the past). If we decide to go back on this decision, maintaining the `--develop` option would provide the least disruption.

Reply:
Ok, makes sense, but at least `--help` output should be updated that this option is currently always on. Also worth mentioning in release notes for the next release.

Reply:
This has been updated to reflect this default setting.